frontend/app/components/Playground.tsx

```diff
@@ -1,21 +1,24 @@
 "use client";
 
-import { Button, Textarea } from "@chakra-ui/react";
+import { Button, Textarea, Heading } from "@chakra-ui/react";
 import { useMutation } from "@tanstack/react-query";
 import React from "react";
 import { runExtraction } from "../utils/api";
 import { Extractor } from "./Extractor";
 import { ResultsTable } from "./ResultsTable";
 
 interface PlaygroundProps {
+  // The playground currently support viewing
+  // both shared and non-shared extractors
```

Comment:
For comments on types/classes/etc use [JSDoc](https://jsdoc.app/about-getting-started) for helpful DX stuff in IDEs
```suggestion
  /**
   * The playground currently support viewing
   * both shared and non-shared extractors
   */
```

Reply:
TIL